packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -1,47 +1,43 @@
 // @ts-check
 
 import { Far } from '@agoric/marshal';
-import { E } from '@agoric/eventual-send';
 import { assert, details as X } from '@agoric/assert';
 import { sameStructure } from '@agoric/same-structure';
 
 import { buildParamManager, ParamType } from '../../../src/paramManager';
 
 const MALLEABLE_NUMBER = 'MalleableNumber';
 
-export const governedParameterTerms = {
-  contractParams: [MALLEABLE_NUMBER],
-};
+const governedParameterTerms = harden([MALLEABLE_NUMBER]);
 
-/* @type {ContractStartFn} */
+/** @type {ContractStartFn} */
 const start = async zcf => {
-  const { electionManager, governedParams } = zcf.getTerms();
-  const governorPublic = E(zcf.getZoeService()).getPublicFacet(electionManager);
+  /** @type {{ governor: Instance, governedParams: Array<string> }} */
+  const { governor, governedParams } = zcf.getTerms();
```

Comment:
The contract that creates this contract would pass in its own instance in the terms here.